### ISSUE TITLE
Terminal not working in version > 0.4.4, 

### ISSUE THREAD

**Issue by vijay-jangir** [jupyter-server/jupyter_server_terminals#103]
## Description
Terminals not working when using version 0.5.0 or 0.5.1, after downgrading to 0.4.4 it works fine.
below is the error.
TypeError: object NoneType can't be used in 'await' expression

## Reproduce
jupyter --versoin screenshot below. should be able to reproduce with jupyter-servier-terminals > 0.4.4
Selected Jupyter core packages...
IPython          : 8.4.0
ipykernel        : 6.28.0
ipywidgets       : not installed
jupyter_client   : 8.6.0
jupyter_core     : 5.6.0
jupyter_server   : 2.12.1
jupyterlab       : 4.0.9
nbclient         : 0.8.0
nbconvert        : 7.13.1
nbformat         : 5.9.2
notebook         : not installed
qtconsole        : not installed
traitlets        : 5.14.0

## Expected behavior
Termianls should work, without any issues.

## Context

- Operating System and version: RHEL 7.9
- Browser and version: Chrome
- Jupyter Server version: 
- Selected Jupyter core packages...
IPython          : 8.4.0
ipykernel        : 6.28.0
ipywidgets       : not installed
jupyter_client   : 8.6.0
jupyter_core     : 5.6.0
jupyter_server   : 2.12.1
jupyterlab       : 4.0.9
nbclient         : 0.8.0
nbconvert        : 7.13.1
nbformat         : 5.9.2
notebook         : not installed
qtconsole        : not installed
traitlets        : 5.14.0

**Comment by vijay-jangir** [comment]
I'm willing to contribute if someone can guide me.

**Comment by danielzgtg** [comment]
> below is the error.
> TypeError

@vijay-jangir What is the full stacktrace?

**Comment by gabelerner** [comment]
@danielzgtg 
```
    Traceback (most recent call last):
      File "/usr/local/lib/python3.10/dist-packages/tornado/ioloop.py", line 750, in _run_callback
        ret = callback()
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 640, in <lambda>
        self.stream.io_loop.add_future(result, lambda f: f.result())
      File "/usr/local/lib/python3.10/dist-packages/jupyter_server_terminals/handlers.py", line 61, in on_message
        await super().on_message(message)
    TypeError: object NoneType can't be used in 'await' expression
    ...
    Traceback (most recent call last):
      File "/usr/local/lib/python3.10/dist-packages/tornado/web.py", line 1790, in _execute
        result = await result
      File "/usr/local/lib/python3.10/dist-packages/jupyter_server_terminals/handlers.py", line 57, in get
        await resp
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 273, in get
        await self.ws_connection.accept_connection(self)
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 863, in accept_connection
        await self._accept_connection(handler)
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 946, in _accept_connection
        await self._receive_frame_loop()
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 1102, in _receive_frame_loop
        await self._receive_frame()
      File "/usr/local/lib/python3.10/dist-packages/tornado/websocket.py", line 1193, in _receive_frame
        await handled_future
    TypeError: object NoneType can't be used in 'await' expression
```

**Comment by gabelerner** [comment]
i checked history and https://github.com/jupyter-server/jupyter_server_terminals/commit/f4df1c6ba7513a2b8f435a950344e668a34c3695#diff-a248e44cf8bffddb8064f63a4068a6d1b5ad336368bf9ab16e0b64d3d9fcd260 seemed to fix it, so i'm going to try `0.5.1`

EDIT: maybe it's https://github.com/jupyter-server/jupyter_server_terminals/blob/main/jupyter_server_terminals/handlers.py#L61 instead that needs a `None` check

**Comment by DrMaphuse** [comment]
I am facing the same issue and I can confirm that `0.5.1` does not fix this issue. Only a downgrade to `0.4.4`.
